paper.tex

```diff
@@ -260,7 +260,65 @@ \subsection*{NEPS revived and roadmap}
 
 \subsection*{Random}
 
-\subsection*{\_\_array\_func\_\_}
+\subsection*{The \_\_array\_function\_\_ Protocol}
+
+The central position which NumPy takes within the python ecosystem with
+respect to scientific computing and numerical works,
+means that a vast number of projects are build on it.
+These projects are consumers of the NumPy API.
+
+On the other side, projects are providers of a
+\emph{NumPy-like API} and array objects targeting audiences with specialized needs beyond
+NumPy's capabilities. Some of these projects provide:
+
+% TODO: A list prints too large currently, so probably should make this
```

Comment:
You could add
```

\newcommand{\tightlist}{\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
```
to the `% New Commands` section around line 31 and then make this
```
\begin{description}
    \tightlist
    \item[SciPy's \code{scipy.sparse} and \code{pydata.sparse}:] Sparse arrays
    \item[CuPy:] Computation on the GPU
    \item[Dask:] Parallel, distributed computing
    \item[PyTorch and TensorFlow:] Deep learning frameworks.
\end{description}
```